src/cart/src/cart/crt_hg.c

```diff
@@ -1115,9 +1115,6 @@ crt_hg_req_send_cb(const struct hg_cb_info *hg_cbinfo)
 				     rpc_pub->cr_ep.ep_rank)) {
 			RPC_TRACE(DB_NET, rpc_priv, "request target evicted\n");
 			rc = -DER_EVICTED;
-		} else if (crt_req_timedout(rpc_priv)) {
-			RPC_TRACE(DB_NET, rpc_priv, "request timedout\n");
-			rc = -DER_TIMEDOUT;
```

Comment:
Does this mean we won't get DER_TIMEDOUT errors returned to caller anymore?

Reply:
Why this case need to check? Usually we cancel RPC after timeout, Nobody cares now about the state it was cancelled.

Reply:
cart caller has ability to manually issue 'cancel' (crt_req_abort/crt_ep_abort) which should trigger crt_hg_req_send_cb(). Currently (outside of tests) raft seems to be the only place where crt_req_abort() is called. It's possible that daos team might need to know whether request was manually cancelled or hit a timeout; would need someone from raft/daos team to verify if it's a problem.

Further searching through daos code there is a lot of handling of DER_TIMEDOUT rc especially in tests; if we no longer return this rc (and return DER_CANCELLED instead), then need to make sure all those places are still working as expected.